src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -886,7 +888,7 @@ private void AddRequestParameter(string name, RequestParameter requestParameter,
                 }
             }
         }
-
+        /*
         [Flags]
         private enum RequestConditionHeaders
```

Comment:
We are adding `RequestConditionHeaders.cs` for enums, can we remove this commented code?

Reply:
removed

Reply:
removed